storage/store/value_delete.go

```diff
@@ -17,6 +17,13 @@ func (b *baseStore) DeletePrefix(ord uint64, prefix string) {
 }
 
 func (b *baseStore) deletePrefix(ord uint64, prefix string) {
+	if _, ok := b.recentlyDeletedPrefixes[prefix]; ok {
+		return
+	}
+	if len(b.recentlyDeletedPrefixes) > 100 {
+		b.recentlyDeletedPrefixes = make(map[string]struct{}) // keep this under reasonable size
+	}
```

Comment:
Maybe we should use a simple cache library, I used some that were very easy to configure, might not be required.

Reply:
I checked, but won't go that route because it is not needed (more complex) and there are many choices, all with their specific performance caveats..